src/edu/gatech/cs2340/trydent/ContinuousEvent.java

```diff
@@ -0,0 +1,59 @@
+package edu.gatech.cs2340.trydent;
+
+/**
+ * Represents an event that is called by the engine every frame.
+ * @author Garrett Malmquist
+ *
+ */
+public abstract class ContinuousEvent {
+	
+	private boolean started = false;
```

Comment:
Whoops. I need to  investigate my IDE settings. That's not supposed to be there =P